platforms/ios/framework/src/TGMapView.mm

```diff
@@ -962,11 +962,8 @@ - (void)touchesBegan:(NSSet *)touches withEvent:(UIEvent *)event {
 - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldRecognizeSimultaneouslyWithGestureRecognizer:(UIGestureRecognizer *)otherGestureRecognizer
 {
     // make shove gesture exclusive
-    if ([gestureRecognizer isKindOfClass:[UIPanGestureRecognizer class]]) {
-        return [gestureRecognizer numberOfTouches] != 2;
-    }
-    if ([otherGestureRecognizer isKindOfClass:[UIPanGestureRecognizer class]]) {
-        return [otherGestureRecognizer numberOfTouches] != 2;
+    if ([gestureRecognizer isEqual:_shoveGestureRecognizer] || [otherGestureRecognizer isEqual:_shoveGestureRecognizer]) {
```

Comment:
`isEqual` seems to work but wouldn't pointer comparison (`==`) do what we want here as well?

Reply:
isEqual: defaults to pointer comparison from `NSObject` when not implemented IIRC, which is why I used it here.